### PR TITLE
Implemented the open ended question and modified some other files

### PR DESCRIPTION
**Pull request by TaherShabaan99**

### CODE REVIEW

**Review by hassanidea — COMMENTED** [review]
Why have you removed QText and Qtype files?

**Review by OmarAgamy9 — APPROVED** [review]
Looks good to me

**Comment by Dez1999** [comment]
Looks good! Good structure and variable names. Just check that the tests are running properly

File: src/main/java/com/SurveyMonkey/model/questions/types/QuestionType.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+public abstract class QuestionType {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    protected int number;
```
Comment (keving3ng) [inline src/main/java/com/SurveyMonkey/model/questions/types/QuestionType.java:11]: id != number, should add another id field

File: src/main/java/com/SurveyMonkey/model/questions/types/QuestionType.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    public QuestionType(String question){
+        this.question = question;
+
+    }
+
+    /**
+     *
+     * @return
+     */
+    public int getId() {
```
Comment (keving3ng) [inline src/main/java/com/SurveyMonkey/model/questions/types/QuestionType.java:34]: update to reflect number/id change

**Review by keving3ng — CHANGES_REQUESTED** [review]

**Pull request closed without merging**